### ISSUE TITLE
Exclude docs and a few other things from the PyPI package

### ISSUE THREAD

**Issue by tleonhardt** [python-cmd2/cmd2#857]
I took a quick look at what is actually in our [latest PyPI package](https://files.pythonhosted.org/packages/03/9d/55b88a06ecc28ba4378b1747311cce30d936325f6f3992ed49e539c56507/cmd2-0.9.24.tar.gz):
```sh
CHANGELOG.md         CONTRIBUTING.md      PKG-INFO             azure-pipelines.yml  cmd2.png             setup.cfg            tests/
CODEOWNERS           LICENSE              Pipfile              cmd2/                docs/                setup.py*            tox.ini
CODE_OF_CONDUCT.md   MANIFEST.in          README.md*           cmd2.egg-info/       examples/            tasks.py
```

Things which probably shouldn't be there include:
* .appveyor.yml
* .coveragerc
* .gitignore
* .travis.yml
* CODE_OF_CONDUCT.md
* Pipfile
* azure-pipelines.yml
* docs/
* tasks.py

These should be excludable by modifying the **MANIFEST.in** file and adding lines like:
```sh
recursive-exclude docs
```

**Comment by deargle** [comment]
I was just looking at this with psiturk last night. I ended up just listing what I wanted to be included: the source directory (`psiturk`), and a few files from the root dir.

https://github.com/NYUCCL/psiTurk/blob/459c09c189406a414ba1c9e57b10511462f00b74/MANIFEST.in
```
recursive-include psiturk *
include requirements.txt
include LICENSE
```

**Comment by tleonhardt** [comment]
@deargle Yeah we had the same approach where we were manually including certain things and I tested that a couple years ago and it worked as intended.  

But somewhere in the interim the behavior changed and a bunch of other stuff was getting put in our PyPI tarball.  I'm not sure if we changed something that altered the behavior or the change is in `setuptools` or what.

I added some explicit **exclude** and **recursive-exclude** statements to our `MANIFEST.in` and it seems to do the trick.
